app/src/main/java/com/example/marcinwisniewski/intivetrainingapp/movielist/model/Movie.kt

```diff
@@ -1,3 +1,32 @@
 package com.example.marcinwisniewski.intivetrainingapp.movielist.model
 
-data class Movie(val id: Int, val title: String, val year: String, val poster: String)
\ No newline at end of file
+import android.os.Parcel
+import android.os.Parcelable
+
+data class Movie(val id: Int, val title: String?, val year: String?, val poster: String?, val genre: String?) : Parcelable {
```

Comment:
I think in this case you can use Serializable instead of Parcelable, just to simplify this class

Reply:
changed